### PR TITLE
Release v2.0.0

### PR DESCRIPTION
**Pull request by pke11y**
Breaking change - only support IP Fabric version v5+

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+## v2.0.0 - 2022-09-22
+
+## Added
+
+- #109 - Migration to use `python-ipfabric` library as the API client to IP Fabric.
+- #103 - New command `get-loaded-snapshots` for greater snapshot details.
+
+## Changed
+
+- Version 2.0.0 of `nautobot-plugin-chatops-ipfabric` only supports IP Fabric version v5.0
```
Comment (chadell) [inline CHANGELOG.md:10]: and higher I guess

Comment (whitej6) [reply CHANGELOG.md:10]: From what I gathered I think each major release on their SDK/OS will require a major release for this plugin. I may have misunderstood though.

**Comment by pauljorgenson** [comment]
As part of this PR you could pass the "timeout" kwarg to the IP Fabric API, currently it's not passed. The two sections below need to be updated

https://github.com/nautobot/nautobot-plugin-chatops-ipfabric/blob/release-v2.0.0/nautobot_chatops_ipfabric/ipfabric_wrapper.py#L100-L104

https://github.com/nautobot/nautobot-plugin-chatops-ipfabric/blob/release-v2.0.0/nautobot_chatops_ipfabric/worker.py#L37-L42

**Comment by justinjeffery-ipf** [comment]
> As part of this PR you could pass the "timeout" kwarg to the IP Fabric API, currently it's not passed. The two sections below need to be updated
> 
> https://github.com/nautobot/nautobot-plugin-chatops-ipfabric/blob/release-v2.0.0/nautobot_chatops_ipfabric/ipfabric_wrapper.py#L100-L104
> 
> https://github.com/nautobot/nautobot-plugin-chatops-ipfabric/blob/release-v2.0.0/nautobot_chatops_ipfabric/worker.py#L37-L42

@pauljorgenson Please update to ipfabric==5.0.11

File: CHANGELOG.md (hunk 1 above)
Comment (pke11y) [reply CHANGELOG.md:10]: correct

Comment (pke11y) [reply CHANGELOG.md]: updated

**Comment by whitej6** [comment]
@justinjeffery-ipf are you wanting the dependency pinned or update in the code referenced?

**Comment by pke11y** [comment]
> @justinjeffery-ipf are you wanting the dependency pinned or update in the code referenced?

Pinned to minimum of latest `ipfabric`

**Review by whitej6 — APPROVED** [review]
LGTM

**Pull request merged**